docs/performance-cluster.md

```diff
@@ -0,0 +1,95 @@
+# Performance Cluster
+Code changes may affect system behavior, scalability, and performance. So we currently have two types of tests needed to validate kubevirt code. These are:
+* Correctness tests - e2e tests verifying expected system behaviors
+* Performance tests - e2e tests to test system performance
+
+Correctness tests run as unit and functional tests for each PR, and performance tests run burst density tests on a dedicated cluster in isolation to continually test builds for performance improvements and regressions. Therefore, performance testing constitutes the final layer of protection against regressions.
+
+
+## Layout
+![infra-layout](infra-layout.svg)
+
+## Prow cluster
+The CI/CD system for performance testing is based on [Prow], a Kubernetes CI system, using the same control plane system as the Correctness tests. Therefore, all test statuses can be viewed in the same testgrid.
+
+However, unlike Correctness tests, Performance testing jobs do not use KubeVirtCI, which creates a Kubernetes cluster in a virtual machine. Instead, a pre-built cluster on top of baremetal nodes is used due to performance reasons. Most speacilly because we want to avoid the performance implications of nested virtualization. Moreover, unlike Correctness tests, only one Performance test job can run at a time to avoid performance interference from workload collocation.
```

Comment:
```suggestion
However, unlike Correctness tests, Performance testing jobs do not use KubeVirtCI, which creates a Kubernetes cluster in a virtual machine. Instead, a pre-built cluster on top of baremetal nodes is used due to performance reasons. Primarily because we want to avoid the performance implications of nested virtualization. Moreover, unlike Correctness tests, only one Performance test job can run at a time to avoid performance interference from workload collocation.
```